codersdk/features.go

```diff
@@ -0,0 +1,29 @@
+package codersdk
+
+type Entitlement string
+
+const (
+	EntitlementEntitled    Entitlement = "entitled"
+	EntitlementGracePeriod Entitlement = "grace_period"
+	EntitlementNotEntitled Entitlement = "not_entitled"
```

Comment:
Instead of `EntitlementEntitled` would `Entitled` suffice? Seems like these are implicit `Entitlement` based on the name.

Reply:
I like the consistency of either always or never including the prefix. It'd be nice if Go would support enums natively one day.

Reply:
I copied 

```
type UserStatus string

const (
	UserStatusActive    UserStatus = "active"
	UserStatusSuspended UserStatus = "suspended"
)
```
for the sake of consistency.  But, if I were writing the package myself for the first time I'd have dropped the prefix.  I don't really mind either way.